beetsplug/bandcamp/metaguru.py

```diff
@@ -77,7 +77,7 @@ def from_html(cls, html: str, config: Optional[JSONDict] = None) -> "Metaguru":
 
     @cached_property
     def excluded_fields(self) -> Set[str]:
-        return set(self.config.get("excluded_fields") or [])
+        return set(self.config.get("exclude_extra_fields") or [])
```

Comment:
Ah, good point, thanks! Could you by any chance to try and add a test for this?

Reply:
I looked into it and as I am unfamiliar with testing and general and the testing setup you have in this repo looks a bit intimidating I am not sure if I can to be honest.

Reply:
You're right, no worries!